src/integrations/watchers/option-wpseo-watcher.php

```diff
@@ -61,6 +62,26 @@ public function check_wincher_option_disabled( $old_value, $new_value ) {
 		return $disabled;
 	}
 
+	/**
+	 * Checks if the WordProof integration is disabled; if so, deletes the tokens
+	 * and website id.
+	 *
+	 * We delete them if the WordProof integration is disabled, no matter if the
+	 * value has actually changed or not.
+	 *
+	 * @param array $old_value The old value of the option.
+	 * @param array $new_value The new value of the option.
+	 *
+	 * @return bool Whether the WordProof tokens have been deleted or not.
+	 */
+	public function check_wordproof_option_disabled( $old_value, $new_value ) {
+		$disabled = $this->check_token_option_disabled( 'wordproof_integration_active', 'wordproof_tokens', $new_value );
+		if ( $disabled ) {
+			\YoastSEO()->helpers->wordproof->remove_site_options();
```

Comment:
Please use the dependency injection instead. See example [here](https://github.com/wordproof/wordpress-seo/blob/wordproof-integration/src/actions/alert-dismissal-action.php). 

```php
	/**
	 * Holds the user helper instance.
	 *
	 * @var User_Helper
	 */
	protected $user;

	/**
	 * Constructs Alert_Dismissal_Action.
	 *
	 * @param User_Helper $user User helper.
	 */
	public function __construct( User_Helper $user ) {
		$this->user = $user;
	}
```